docs/secure-connections/sauce-connect-5/guides/localhost-proxying.md

```diff
@@ -0,0 +1,62 @@
+---
+id: localhost-proxying
+title: Proxying Localhost with Sauce Connect 5
+sidebar_label: Proxying Localhost
+---
+
+Sauce Connect 5 enables your local applications and services to be securely accessed from the Sauce Labs cloud. However, special care is required when working with `localhost` addresses, particularly when testing from mobile devices or when using real devices in Sauce Labs.
+
+## Enabling Localhost Proxying
+
+To route localhost traffic through Sauce Connect, you must enable the [`--proxy-localhost`](/dev/cli/sauce-connect-5/run#proxy-localhost) option when starting the tunnel.
+
+This option controls how Sauce Connect handles requests to localhost (or 127.0.0.1):
+
+- `--proxy-localhost allow`: Proxies all localhost requests through Sauce Connect to your local machine. Use this mode when you want devices in Sauce Labs to access services running on your machine (for example, http://localhost:3000).
+- `--proxy-localhost direct`: Allows localhost requests but skips proxying them through any configured upstream proxy. Use this your upstream proxy localhost should not be exposed.
+- `--proxy-localhost deny`: Blocks all localhost requests to prevent unintended exposure. It is the safest option when localhost access is not needed.
+
+By default Sauce Connect 5 uses deny mode.
+
+## Special Cases
+
+In some environments - particularly when testing on real mobile devices - localhost requires special handling.
+That is because devices and browsers treat localhost or 127.0.0.1 as referring to themselves, not to the machine running Sauce Connect.
+
+### Configuring Mobile Devices For Testing `localhost`
+
+Testing with the address `localhost` (or the IP address `127.0.0.1`) is not supported with iOS or Android real devices in Sauce Connect.
+
+To work around this, you'll need to edit your `hosts` file on the machine on which you are running Sauce Connect. Add an entry for a placeholder hostname (such as `localtestsite`) and the IP address `127.0.0.1`. Requests for `localtestsite` in your tests will then be sent through your Sauce Connect tunnel to `localhost`, which is the machine on which you are running Sauce Connect.
+
+For example, adding `127.0.0.1   mockserver` to your `/etc/hosts` file, then starting a server on `localhost:3333` will route mockserver:3333 HTTP calls to your local server. Mobile tests using Sauce Connect will then be able to find your local server regardless of the nature of your test.
```

Comment:
maybe to avoid confusion let's use `127.0.0.1:3333` instead of `localhost:3333`

Reply:
Good idea.